docs/docs/Configuration/environment-variables.md

```diff
@@ -375,3 +375,97 @@ To run Langflow using the above VSCode `tasks.json` file, in the VSCode command
 
 </TabItem>
 </Tabs>
+
+## Set environment variables for Langflow Desktop
+
+Environment variables set in your terminal are not automatically available to GUI-based applications like Langflow Desktop when launched through the Finder or the Start Menu.
+
+The configuration of environment variables differs between macOS and Windows installations.
```

Comment:
We haven't published docs about the Windows version of Langflow Desktop yet. Are you planning to wait to merge this PR until we have content for installing the Windows version?

Reply:
Good call. Added DO NOT MERGE tag.